### PR TITLE
PhpClosingTagFixer - simplify flow, no need for loop

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: Symfony/CS/Fixer/PSR2/PhpClosingTagFixer.php [hunk 1]
```diff
@@ -35,16 +35,21 @@ public function fix(\SplFileInfo $file, $content)
...
         $closeTags = $tokens->findGivenKind(T_CLOSE_TAG);
 
-        foreach (array_reverse($closeTags, true) as $index => $token) {
-            $tokens->removeLeadingWhitespace($index);
-            $token->clear();
+        if (empty($closeTags)) {
+            return $content;
+        }
+
+        list($index, $token) = each($closeTags);
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/PhpClosingTagFixer.php]: We don't need $token to be defined here.

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/PhpClosingTagFixer.php]: now we need it

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/PhpClosingTagFixer.php]: nice :)

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request closed without merging**